content/about/feature-stages/index.md

```diff
@@ -30,23 +30,22 @@ Below is our list of existing features and their current phases. This informatio
 
 | Feature           | Phase
 |-------------------|-------------------
-| [Protocols: HTTP 1.1](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/http_connection_management.html#http-protocols)  | Beta
-| [Protocols: HTTP 2.0](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/http_connection_management.html#http-protocols)  | Alpha
-| [Protocols: gRPC](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/grpc)   | Alpha
+| [Protocols: HTTP 1.1](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/http_connection_management.html#http-protocols)  | Stable
+| [Protocols: HTTP 2.0](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/http_connection_management.html#http-protocols)  | Beta
+| [Protocols: TCP](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/tcp_proxy)   | Beta
+| [Protocols: gRPC](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/grpc)   | Beta
+| [Protocols: Websocket](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/websocket)   | Beta
 | [Protocols: MongoDB](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/mongo)      | Alpha
-| [Request Routing](/docs/tasks/traffic-management/request-routing/)      | Alpha
-| [Fault Injection](/docs/tasks/traffic-management/fault-injection/)      | Alpha
-| [Traffic Shifting](/docs/tasks/traffic-management/traffic-shifting/)      | Alpha
-| [Circuit Break](/docs/tasks/traffic-management/circuit-breaking/)      | Alpha
-| [Mirroring](/docs/tasks/traffic-management/mirroring/)      | Alpha
-| [Ingress Traffic](/docs/tasks/traffic-management/ingress/)      | Alpha
+| [Request Routing](/docs/tasks/traffic-management/request-routing/)      | Beta
+| [Fault Injection](/docs/tasks/traffic-management/fault-injection/)      | Beta
+| [Traffic Shifting](/docs/tasks/traffic-management/traffic-shifting/)      | Beta
+| [Circuit Break](/docs/tasks/traffic-management/circuit-breaking/)      | Beta
+| [Mirroring](/docs/tasks/traffic-management/mirroring/)      | Beta
+| [Ingress Traffic](/docs/tasks/traffic-management/ingress/)      | Beta
```

Comment:
I tried to clean all of this up in my other PR. These don’t make sense for the end user. We need to be concrete and avoid inconsistency.